hazelcast/src/main/java/com/hazelcast/nio/Address.java

```diff
@@ -34,7 +33,6 @@
 /**
  * Represents an address of a member in the cluster.
  */
-@BinaryInterface
```

Comment:
Is there a decision change about the usage of BinaryInterface? I am probably missing info here.

Reply:
Since Address already has `AddressCodec`, we do not need this marker anyway. Vassilis suggested this and some others as well.

Reply:
Ok, I am reverting this change so that this can be handled in another PR.